tt_metal/tt_metal.cpp

```diff
@@ -319,61 +319,12 @@ std::map<chip_id_t, Device *> CreateDevices(
 }
 
 void CloseDevices(std::map<chip_id_t, Device *> devices) {
```

Comment:
I always thought this was an awkward interface because it also expects `devices` to be ordered in a specific way. Could we just change this directly to pass an ordered list of devices?

Reply:
I don't expect this to happen in this PR, to be clear, unless it's easy to do.

Reply:
It's a convenience thing, since CreateDevices passes back the map. So we have test fixtures that takes what the CreateDevices returns and passes it back into CloseDevices.